app/models/alchemy/element.rb

```diff
@@ -107,20 +144,32 @@ def create_from_scratch(attributes)
       #
       # == Example
       #
-      #   @copy = Alchemy::Element.copy(@element, {:public => false})
+      #   @copy = Alchemy::Element.copy(@element, {public: false})
       #   @copy.public? # => false
       #
-      def copy(source, differences = {})
-        source.attributes.stringify_keys!
+      def copy(source_element, differences = {})
+        source_element.attributes.stringify_keys!
         differences.stringify_keys!
-        attributes = source.attributes.except(*SKIPPED_ATTRIBUTES_ON_COPY).merge(differences)
-        element = self.create!(attributes.merge(:create_contents_after_create => false))
-        element.tag_list = source.tag_list
-        source.contents.each do |content|
-          new_content = Content.copy(content, :element_id => element.id)
-          new_content.move_to_bottom
+
+        attributes = source_element.attributes
+                       .except(*SKIPPED_ATTRIBUTES_ON_COPY)
```

Comment:
Style/MultilineOperationIndentation: Use 2 (not 15) spaces for indenting an expression in an assignment spanning multiple lines.